apps/gp2-server/src/data-providers/working-group.data-provider.ts

```diff
@@ -32,15 +56,53 @@ export class WorkingGroupSquidexDataProvider
     return {
       total: result.queryWorkingGroupsContentsWithTotal.total,
       items: result.queryWorkingGroupsContentsWithTotal.items.map(
-        (workingGroup) => ({
-          ...workingGroup.flatData,
-          id: workingGroup.id,
-          title: workingGroup.flatData.title || '',
-          shortDescription: workingGroup.flatData.shortDescription || '',
-          leadingMembers: workingGroup.flatData.leadingMembers || '',
-          members: [],
-        }),
+        parseWorkingGroupToDataObject,
       ),
     };
   }
 }
+type Member = NonNullable<GraphQLWorkingGroup['flatData']['members']>[number];
```

Comment:
I think if you use this snippet (and another one below) you can get rid of all this type craziness at the top:
```suggestion
type Member = NonNullable<
  NonNullable<
    NonNullable<
      NonNullable<
        NonNullable<FetchWorkingGroupsQuery>['queryWorkingGroupsContentsWithTotal']
      >['items']
    >[number]['flatData']
  >['members']
>[number];
```

Reply:
done